Protocol.md

```diff
@@ -228,10 +234,25 @@ The recipient:
   - on receipt of a valid `sign_dlc`:
     - SHOULD broadcast the funding transaction.
 
+### Script Pubkey Standardness Definition
+
+ For a script pub key to be valid it must be in one of the following forms:
+
+    1. `OP_DUP` `OP_HASH160` `20` 20-bytes `OP_EQUALVERIFY` `OP_CHECKSIG` (pay to pubkey hash), OR
+    2. `OP_HASH160` `20` 20-bytes `OP_EQUAL` (pay to script hash), OR
+    3. `OP_0` `20` 20-bytes (version 0 pay to witness pubkey hash), OR
+    4. `OP_0` `32` 32-bytes (version 0 pay to witness script hash), OR
+    5. `OP_1` through `OP_16` inclusive, followed by a single push of 2 to 40 bytes
+       (witness program versions 1 through 16)
+
+  These script pub key forms include only standard forms accepted by the Bitcoin network, which ensures the resulting transaction will propagate to miners.
```

Comment:
I would phrase this differently, "accepted by the wider set of deployed Bitcoin clients in the network, which increase the chances of successful propagation to miners". Policy rules are heterogeneous by design.